test/example.md

```diff
@@ -167,6 +167,17 @@ callAsyncFunction((a, b) => {
 });
 ```
 
+## functionWithFunction(fn, number)
+
+- `fn`: [`<Function>`][function] function description
+  - `num`: [`<number>`][number] `fn` parameter
+- _Returns:_ [`<number>`][number]
```

Comment:
Wouldn't it be better to remove the dash here to avoid grouping Returns of a function with other arguments?
```suggestion
  _Returns:_ [`<number>`][number]
```

Reply:
Without dash, it either has to have no indentation and empty lines before and after because of prettier which won't look nice as there may be multiple nested functions.
```md
- `fn`: [`<Function>`][function] function description
  - `num`: [`<number>`][number] `fn` parameter

_Returns:_ [`<number>`][number]

- `number`: [`<number>`][number]
```
With indentation we will need to add an empty line after each line because of prettier.
```md
- `fn`: [`<Function>`][function] function description

  - `num`: [`<number>`][number] `fn` parameter

   _Returns:_ [`<number>`][number]

- `number`: [`<number>`][number]
```
or it will not render properly as `_Returns:_` will be moved to the previous line.
```md
- `fn`: [`<Function>`][function] function description
  - `num`: [`<number>`][number] `fn` parameter
      _Returns:_ [`<number>`][number]
- `number`: [`<number>`][number]
```

Reply:
Wouldn't it be fine like this
```
- `fn`: [`<Function>`][function] function description
  - `num`: [`<number>`][number] `fn` parameter
    _Returns:_ [`<number>`][number]
- `number`: [`<number>`][number]
```
IF we put two spaces at the end of ``  - `num`: [`<number>`][number] `fn` parameter  ``?

Anyway @belochub @nechaido WDYT?

Reply:
Yes, it will help with it being on the previous line, but it will be rendered as if it was on the same level as the last argument. And since they can be nested

```md
- `fn`: [`<Function>`][function] function description
  - `obj`: [`<Object>`][object] `fn` parameter asd
    - `num`: [`<number>`][number] `fn` parameter asd  
  _Returns:_ [`<number>`][number]
- `number`: [`<number>`][number]
```
will be rendered as

- `fn`: [`<Function>`][function] function description
  - `obj`: [`<Object>`][object] `fn` parameter asd
    - `num`: [`<number>`][number] `fn` parameter asd  
  _Returns:_ [`<number>`][number]
- `number`: [`<number>`][number]

[object]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object
[function]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function
[number]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#Number_type

Reply:
Perhaps then we should at least indent Returns same way as function arguments to avoid grouping it with the rest of the arguments to the main function?
```md
- `fn`: [`<Function>`][function] function description
  - `obj`: [`<Object>`][object] `fn` parameter
    - `num`: [`<number>`][number] `fn`
  -  _Returns:_ [`<number>`][number]
- `number`: [`<number>`][number]
```

Reply:
It seems that the empty line before the first `Returns:` in the third example was not rendered. Anyway, md generation like in the 1st and 2nd example means that levels in code and in docs would be different which can lead to some confusion. As for the third one, IMO extra empty lines take too much space, though if we can get rid of them it will look better than the current version.

Reply:
I couldn't get md to render an empty line after a list (unless `<br />` tag was used) so it's not usable I guess.